terraform/azure/variables.tf

```diff
@@ -568,6 +568,18 @@ variable "sbd_storage_type" {
 # If iscsi is selected as sbd_storage_type
 # Use the next variables for advanced configuration
 
+variable "iscsi_count" {
+  description = "The number of iscsi servers to deploy"
+  type        = number
+  default     = 1
+  validation {
+    condition = (
+      var.iscsi_count >= 1 && var.iscsi_count <= 3
```

Comment:
My thought was that if you enable iscsi by setting at least one of the fencing mechanisms to 'sbd' you can then also set how many with this value.  If you don't set a value, you'll still get one iscsi server by default.  If you set all the fencing mechanisms to 'native' you won't get any iscsi at all.  I see this as a variable that controls the number of servers when enabled rather than a switch for enabling iscsi as there is already a method for that.